airbyte-integrations/connectors/source-freshcaller/source_freshcaller/streams.py

```diff
@@ -126,15 +126,15 @@ def stream_slices(self, stream_state: Mapping[str, Any] = None, **kwargs) -> Ite
             ...]
         """
         start_date = pendulum.parse(self.start_date).in_timezone("UTC")
-        end_date = pendulum.utcnow().subtract(minutes=self.sync_lag_minutes)  # have a safe lag
+        end_date = pendulum.now("UTC").subtract(minutes=self.sync_lag_minutes)  # have a safe lag
```

Comment:
I've bumped `airbyte-cdk` to `~=0.2` and removed pendulum in `setup.py`. 
In Pendulum `utcnow()` is [deprecated](https://pendulum.eustace.io/history/) since 2.0.0